bots/github/__init__.py

```diff
@@ -277,22 +277,38 @@ def pulls(self):
                 count = len(pulls)
         return result
 
-    def issues(self, labels=[ "bot" ], state="open"):
+    def issues(self, labels=[ "bot" ], state="open", after=None):
```

Comment:
Since `after` can also be used in the sense of a sequence, I recommend either renaming it to `since` and/or adding a comment. A comment might not be bad, since this time is either creation or closure time, depending on the issue state.